src/components/block-tunes/block-tune-move-down.ts

```diff
@@ -84,6 +84,11 @@ export default class MoveDownTune implements BlockTune {
     }
 
     const nextBlock = this.api.blocks.getBlockByIndex(currentBlockIndex + 1);
+
+    if (!nextBlock) {
```

Comment:
how it can happen? This statement is checking above.

Reply:
idk